src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,47 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```

위처럼 표현한다면 소소하지만 가독성을 향상 시킬 수 있을 것 같아요.

Reply:
그렇군요.. 감사합니다!